src/DepOutputChannel.ts

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+import * as vscode from 'vscode';
+
+export class DepOutputChannel {
+  outputChannel: vscode.OutputChannel;
+  constructor(channelName = 'Dependency Analytics') {
```

Comment:
add functionality to get the `Dependency Analytics` command from the command palette.

Reply:
i didn't get you, it's just the Output channel default name, what it has to with Command palette.

Reply:
what if I want to see the `Dependency Analytics` log from command Palette?